code/modules/antagonists/morph/morph.dm

```diff
@@ -47,7 +47,7 @@
 							You may take the form of anything nearby by shift-clicking it. This process will alert any nearby \
 							observers, and can only be performed once every five seconds. While morphed, you move faster, but do \
 							less damage. In addition, anyone within three tiles will note an uncanny wrongness if examining you. \
-							You can attack any item or dead creature to consume it - creatures will fully restore your health. \
+							You can attack any item or dead creature to consume it - creatures will restore your health. \
```

Comment:
Eating bodies heals 50, the morph's health is 150. Maybe I'm wrong and it's doing something elsewhere in the code but it seems they aren't actually fully healing.